scc/delivery-with-flux.hbs.md

```diff
@@ -1,212 +1,208 @@
 # Use Gitops Delivery with FluxCD (alpha)
 
-This topic outlines the delivery of packages created by the Out of the Box Supply Chain
-with `carvel-package-workflow-enabled` and added to a GitOps repository to one or more run clusters.
+This topic explains how to deliver Carvel `Packages`, created by the Carvel Package Supply Chains, from a GitOps repository to one or more run clusters using FluxCD.
 
 ## Prerequisites
 
-To use Gitops Delivery with FluxCD, complete the following prerequisites:
+To use Gitops Delivery with FluxCD, you must complete the following prerequisites:
 
-- The Build cluster is created and has network access to your run clusters. It must have FluxCD Kustomize Controller installed. See the [Flux documentation](https://fluxcd.io/flux/installation/#dev-install).
-- Run clusters serve as your deploy environments. They can either be Tanzu Application Platform clusters, or regular Kubernetes clusters, but they must have kapp-controller installed. See the [Carvel documentation](https://carvel.dev/kapp-controller/).
-- To target run clusters from the Build cluster, you must create a secret containing each run cluster Kubeconfig. The FluxCD Kustomization resource uses these secrets.
+- You must create a `Workload` that uses the Carvel Package Supply Chains. See the [documentation](./carvel-package-supply-chain.hbs.md). You must have at least one Carvel `Package` generated by this `Workload` stored in your GitOps repository.
+- You must have at least one Run cluster. Run clusters serve as your deployment environments. They can either be Tanzu Application Platform clusters, or regular Kubernetes clusters, but they must have kapp-controller and Contour installed. See the [Carvel documentation](https://carvel.dev/kapp-controller/) and the [Contour documentation](https://projectcontour.io/).
+- You must create a Build cluster that has network access to your Run clusters. It must have FluxCD Kustomize Controller installed. See the [Flux documentation](https://fluxcd.io/flux/installation/#dev-install).
 
```

Comment:
In the preview I see this line twice, but here its seems correct, not sure why

Reply:
Might have something to do with the force push I did